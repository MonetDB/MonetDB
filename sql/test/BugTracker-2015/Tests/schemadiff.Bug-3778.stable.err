stderr of test 'schemadiff.Bug-3778` in directory 'sql/test/BugTracker-2015` itself:


# 15:29:05 >  
# 15:29:05 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33909" "--set" "mapi_usock=/var/tmp/mtest-7442/.s.monetdb.33909" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/ufs/manegold/_/Monet/HG/default/prefix/_/var/MonetDB/mTests_sql_test_BugTracker-2015" "--set" "mal_listing=0" "--set" "embedded_r=yes"
# 15:29:05 >  

# builtin opt 	gdk_dbpath = /ufs/manegold/_/Monet/HG/default/prefix/_/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 33909
# cmdline opt 	mapi_usock = /var/tmp/mtest-7442/.s.monetdb.33909
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbpath = /ufs/manegold/_/Monet/HG/default/prefix/_/var/MonetDB/mTests_sql_test_BugTracker-2015
# cmdline opt 	mal_listing = 0
# cmdline opt 	embedded_r = yes
# cmdline opt 	gdk_debug = 536870922

# 15:29:05 >  
# 15:29:05 >  "/usr/bin/python2" "schemadiff.Bug-3778.SQL.py" "schemadiff.Bug-3778"
# 15:29:05 >  

# builtin opt 	gdk_dbpath = /home/niels/scratch/monetdb/Linux-x86_64/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_usock = /var/tmp/mtest-25081/.s.monetdb.33961
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	mapi_port = 37355
# cmdline opt 	gdk_dbpath = /tmp/tmpou_du0/master/master
# cmdline opt 	gdk_debug = 536870922
# builtin opt 	gdk_dbpath = /home/niels/scratch/monetdb/Linux-x86_64/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_usock = /var/tmp/mtest-25081/.s.monetdb.33961
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	mapi_port = 48845
# cmdline opt 	gdk_dbpath = /tmp/tmpou_du0/worker_0/worker_0
# cmdline opt 	gdk_debug = 536870922

# 11:56:09 >  
# 11:56:09 >  "Done."
# 11:56:09 >  

