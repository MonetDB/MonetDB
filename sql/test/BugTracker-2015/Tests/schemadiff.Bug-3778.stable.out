stdout of test 'schemadiff.Bug-3778` in directory 'sql/test/BugTracker-2015` itself:


# 15:29:05 >  
# 15:29:05 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33909" "--set" "mapi_usock=/var/tmp/mtest-7442/.s.monetdb.33909" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/ufs/manegold/_/Monet/HG/default/prefix/_/var/MonetDB/mTests_sql_test_BugTracker-2015" "--set" "mal_listing=0" "--set" "embedded_r=yes"
# 15:29:05 >  

(mapi:monetdb://monetdb@localhost/worker_0) Parameter 1 has wrong SQL type, expected int(32), but got bigint(64) instead


# 10:44:31 >  
# 10:44:31 >  "Done."
# 10:44:31 >  

